test/integration/image-future/typescript/next.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   images: {
     domains: ['via.placeholder.com'],
-    // disableStaticImages: true,
+    disableStaticImages: true,
```

Comment:
Was this change intended?

Reply:
Nice catch, fixed!

(must have been changed by running tests locally)